MyUnitTests/MyTargetClassTests.cs

```diff
@@ -24,5 +24,26 @@ public void VerifySubtraction()
             var intResult = objTargetClass.SubractNumbers(intFirstNumber, intsecondNumber);
             Assert.Equal(15 - 10, intResult);
         }
+
+        [Fact]
+        public void VerifyDivide()
+        {
+            const int intFirstNumber = 15;
+            const int intsecondNumber = 10;
+            var objTargetClass = new MyTargetClass();
+            var intResult = objTargetClass.DivideNumbers(intFirstNumber, intsecondNumber);
+            Assert.Equal(15 / 10, intResult);
+        }
+
+        [Fact]
+        public void VerifyMultiple()
+        {
+            const int intFirstNumber = 15;
```

Comment:
Please consider using class level constant hence same constants are being used in multiple methods.